tools/emoji_dump/build_emoji

```diff
@@ -4,4 +4,4 @@ rm -rf static/third/gemoji/images
 (cd tools/emoji_dump && python ./emoji_dump.py)
 mkdir -p static/third/gemoji/images
 mv tools/emoji_dump/out static/third/gemoji/images/emoji
-cp -a assets/zulip-emoji/* static/third/gemoji/images/emoji/
+cp -RPp assets/zulip-emoji/* static/third/gemoji/images/emoji/
```

Comment:
Just to confirm, does -RPp preserve symlinks on OpenBSD?  If so then this part lgtm.